### PR TITLE
New package: ufetch-1

### PR DESCRIPTION
**Pull request by Kozova1**

### CODE REVIEW

**Comment by Kozova1** [comment]
well that worked

**Comment by teldra** [comment]
If you pushed your commit to origin, and make changes afterwards, it would be better, even mandatory iirc, to ```git commit --amend; git push --force origin``` your additions.
And make a new branch from master for every package, not from a branch or you end up mixing them.

File: srcpkgs/ufetch/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+pkgname=ufetch
+version=1
+revision=1
+archs="noarch"
+build_style=fetch
+short_desc="Tiny system info for Unix-like operating systems"
+maintainer="Kozova1 <mug66kk@gmail.com>"
+license="ISC"
+homepage="https://gitlab.com/jschx/ufetch"
+distfiles="https://gitlab.com/jschx/ufetch/raw/master/ufetch-void https://gitlab.com/jschx/ufetch/raw/master/LICENSE"
```
Comment (Duncaen) [inline srcpkgs/ufetch/template]: This should use a tagged release or a fixed commit.

Comment (Duncaen) [inline srcpkgs/ufetch/template]:
```suggestion
distfiles="https://gitlab.com/jschx/ufetch/raw/master/ufetch-void
 https://gitlab.com/jschx/ufetch/raw/master/LICENSE"
checksum="53af7955e2bfa974447243dd22526a9763cfe65f9ab04002fe572797358c9177
 ca15acfdbb3790cf0cb094528e3e16fe411be20d006cb3e623687306908e0621"
```

**Comment by Kozova1** [comment]
> If you pushed your commit to origin, and make changes afterwards, it would be better, even mandatory iirc, to `git commit --amend; git push --force origin` your additions.
> And make a new branch from master for every package, not from a branch or you end up mixing them.
> Now, you can squash your commits into one and force push them. Read about it. ;)

I think I gotcha

File: srcpkgs/ufetch/template (hunk 1 above)
Comment (Kozova1) [reply srcpkgs/ufetch/template]: both are done

File: srcpkgs/ufetch/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'ufetch'
+pkgname=ufetch
+version=ee7b9b1e32956b046c3fb7c8f199f97e3b95a58c
```
Comment (maxice8) [inline srcpkgs/ufetch/template]: that is not an actual version

File: srcpkgs/ufetch/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+revision=1
+archs="noarch"
```
Comment (maxice8) [inline srcpkgs/ufetch/template]: no need to be quoted

**Comment by maxice8** [comment]
https://github.com/voidlinux/void-packages/pull/11708

**Comment by dawidpotocki** [comment]
> Ask upstream to tag a release.

It is tagged.

File: srcpkgs/ufetch/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+build_style=fetch
+short_desc="Tiny system info for Unix-like operating systems"
+maintainer="Kozova1 <mug66kk@gmail.com>"
+license="ISC"
+homepage="https://gitlab.com/jschx/ufetch"
+distfiles="https://gitlab.com/jschx/ufetch/raw/${version}/ufetch-void
+ https://gitlab.com/jschx/ufetch/raw/${version}/LICENSE"
+checksum="53af7955e2bfa974447243dd22526a9763cfe65f9ab04002fe572797358c9177
+ca15acfdbb3790cf0cb094528e3e16fe411be20d006cb3e623687306908e0621"
```
Comment (maxice8) [inline srcpkgs/ufetch/template]: 1 whitespace prefix for newline when breaking checksum= into multiple lines

**Pull request closed without merging**